### PR TITLE
feat(an): Information about institue/company/association screens added

### PR DESCRIPTION
**Pull request by Thorrik58**
# Information about institue/company/association screens added

Attach a link to issue if relevant

## What

Adding information about institue/company/association screens

## Why

User needs to specify which company is related to the accident

## Screenshots / Gifs
School
image

Fishery
image

Sport club
image

Rescue squad
image

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

File: libs/application/templates/accident-notification/src/forms/AccidentNotificationForm.ts [hunk 1]
```diff
@@ -441,7 +447,14 @@ export const AccidentNotificationForm: Form = buildForm({
               titleVariant: 'h5',
               title: companyInfo.labels.descriptionField,
               condition: (formValue) => {
-                return formValue.isRepresentativeOfCompany?.toString() === YES
+                console.log(formValue)
```
Comment (sigruntg) [inline libs/application/templates/accident-notification/src/forms/AccidentNotificationForm.ts:450]: Two console logs

File: libs/application/templates/accident-notification/src/lib/messages/rescueSquadInfo.ts [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    },
+    title: {
+      id: 'an.application:rescueSquadInfo.general.title',
+      defaultMessage: 'Upplýsingar um björgunarsveit',
+      description: 'Information about rescue squad',
+    },
+    description: {
+      id: 'an.application:rescueSquadInfo.general.description',
+      defaultMessage:
+        'Vinsamlegast fylltu út upplýsingar um björgunarsveitina..',
```
Comment (sigruntg) [inline libs/application/templates/accident-notification/src/lib/messages/rescueSquadInfo.ts:18]: Two dots

**Review by sigruntg — CHANGES_REQUESTED** [review]
Two minor issues.

File: libs/application/templates/accident-notification/src/lib/messages/rescueSquadInfo.ts (hunk 2 above)
Comment (Thorrik58) [reply libs/application/templates/accident-notification/src/lib/messages/rescueSquadInfo.ts]: Good eye, will fix 🙌

File: libs/application/templates/accident-notification/src/forms/AccidentNotificationForm.ts (hunk 1 above)
Comment (Thorrik58) [reply libs/application/templates/accident-notification/src/forms/AccidentNotificationForm.ts]: Nice catch, will fix 🙌

**Review by sigruntg — APPROVED** [review]
Looks good!

**Pull request merged**